packages/grid/data-grid/src/tests/pagination.DataGrid.test.tsx

```diff
@@ -320,6 +320,34 @@ describe('<DataGrid /> - Pagination', () => {
         );
         expect(getColumnValues(0)).to.deep.equal(['7', '8']);
       });
+
+      it('should display a warning if the pageSize is not in the rowsPerPageOptions', () => {
+        const pageSize = 12;
+
+        const TestCase: React.FC = () => {
```

Comment:
Just a small thing. We stopped using `React.FC` in #1325.

```suggestion
        const TestCase = () => {
```

Reply:
We can use `React.VoidFunctionComponent` to exclude the children